cq_editor/main_window.py

```diff
@@ -46,8 +46,11 @@ def __init__(self,parent=None):
 
         self.prepare_statusbar()
         self.prepare_actions()
-        
-        self.components['object_tree'].addLines()
+
```

Comment:
The application (without bundling) did work on Mac (according to some users). So I'd rather not remove this functionality but rather fix the graphics initialization process

Reply:
If I leave this statement, the resulting app binary is guaranteed to crash (verified in macOS 10.14 and 10.15).  We can either defer this pull request until a root cause is found or leave the 'if platform' guard in place until a root cause is found for future releases.  Unless there is some urgency to make a new release including macOS, we can always defer the PR and wait until the root cause / remedy to this race condition crash at startup.

Reply:
Actually if you are willing to help, I have some ideas regarding debugging this.

Reply:
Sounds good.  I did spend a few hours yesterday trying to get more clues.  One issue is the timing when “InitDriver” is called relative to MainWindow.show().  The rest of the issue is related to abi trap from AIS interactive viewer.  I think it tries to throw an exception and it doesn’t get handled properly.

Reply:
@adam-urbanczyk let me know if you want to proceed with some macOS debugging of this issue--I would like to help if I can.  BTW, good luck with the FOSDEM talk--I just checked out your slide deck!